Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -442,7 +443,7 @@ await useCase.Execute((organisationId, removePersonFromOrganisation))
                 return operation;
             });
 
-        app.MapGet("/{organisationId}/persons/person-invites",
+        app.MapGet("/{organisationId}/invites",
```

Comment:
@shilpigoel-goaco Why are you refactoring this? This will be a breaking change for the front end!

Reply:
This change has been suggested by Kuba